openssl/src/x509/mod.rs

```diff
@@ -228,7 +228,7 @@ impl X509Builder {
     /// the X.509 standard should pass `2` to this method.
     #[corresponds(X509_set_version)]
     pub fn set_version(&mut self, version: i32) -> Result<(), ErrorStack> {
-        unsafe { cvt(ffi::X509_set_version(self.0.as_ptr(), version.into())).map(|_| ()) }
+        unsafe { cvt(ffi::X509_set_version(self.0.as_ptr(), version as c_long)).map(|_| ()) }
```

Comment:
Why?

Reply:
Clippy emits a warning, if `.into()` and the target type is the same as the source type:
```
warning: useless conversion to the same type: `i32`
   --> openssl\src\x509\mod.rs:231:61
    |
231 |         unsafe { cvt(ffi::X509_set_version(self.0.as_ptr(), version.into())).map(|_| ()) }
    |                                                             ^^^^^^^^^^^^^^ help: consider removing `.into()`: `version`
    |
    = note: `#[warn(clippy::useless_conversion)]` on by default
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#useless_conversion
```